File: ComiAPI/ComiCore/Model/Actor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ComiCore.Model
{
    public class Actor : BaseEntity
    {
        public string Name { get; set; }
        public string Photo { get; set; }
        public virtual ICollection<MoviesActor> MoviesActors { get; set; }
    }
}

File: ComiMovies/src/app/models/actor.ts
import { BaseEntity } from './ientity';

export class Actor extends BaseEntity {
    name: string
    photo: string
    moviesActors: Array<any>

    constructor(id:number, name:string, photo:string){
        super();
        this.id = id;
        this.name = name;
        this.photo = photo;
    }
}
